python/kfserving/docs/KFServingClient.md

```diff
@@ -20,6 +20,8 @@ KFServingClient | [set_credentials](#set_credentials) | Set Credentials|
 KFServingClient | [create](#create) | Create KFService|
 KFServingClient | [get](#get)    | Get or watch the specified KFService or all KFServices in the namespace |
 KFServingClient | [patch](#patch)  | Patch the specified KFService|
+KFServingClient | [replace](#replace) | Replace the specified KFService|
+KFServingClient | [promote](#promote) | Promote the specified KFService|
```

Comment:
Promote the canary version of the KFService to default

Reply:
so cool :-)

Reply:
Updated, thanks!